src/router/handlers/wellKnownUriHandler.test.ts

```diff
@@ -0,0 +1,68 @@
+import { SmartStrategy } from 'fhir-works-on-aws-interface';
+import { camelToSnakeCase, getWellKnownUriResponse } from './wellKnownUriHandler';
+
+describe('camelToSnakeCase', () => {
+    test('one word', () => {
+        expect(camelToSnakeCase('red')).toEqual('red');
+    });
```

Comment:
Add a mis-capitalized test case
```suggestion
    test('one word', () => {
        expect(camelToSnakeCase('red')).toEqual('red');
    });
     test('one word; mis-capitalized ', () => {
        expect(camelToSnakeCase('Red')).toEqual('red');
    });
```

Reply:
Added this test case, and fixed the code to handle this edge case.